packages/do-not-merge/src/do-not-merge.ts

```diff
@@ -78,7 +78,7 @@ export = (app: Probot) => {
         }
         return;
       }
-      logger.info('Do not merge label found');
+      logger.metric('do_not_merge.add_label');
```

Comment:
Might want to move this down, right next to the `Creating failed check` log. That way, each PR is only counted once.

Reply:
done 👍